File: java/Clothing.java

public class Clothing extends Product {
    private String size;
    private String material;
    private String gender;

    public Clothing() {

    }

    public Clothing(String size, String material, String gender) {
        this.size = size;
        this.material = material;
        this.gender = gender;
    }

    // setter gettetr

    public String getSize() {
        return this.size;
    }

    public void setSize(String size) {
        this.size = size;
    }

    public String getMaterial() {
        return this.material;
    }

    public void setMaterial(String material) {
        this.material = material;
    }

    public String getGender() {
        return this.gender;
    }

    public void setGender(String gender) {
        this.gender = gender;
    }
}

File: python/Clothing.py
from Product import Product


class Clothing(Product):
    __size = ""
    __material = ""
    __gender = ""

    # contructor
    def __init__(self, size, material, gender):
        self.__size = size
        self.__material = material
        self.__gender = gender

    # getter setter

    def getSize(self):
        return self.__size

    def setSize(self, size):
        self.__size = size

    def getMaterial(self):
        return self.__material

    def setMaterial(self, material):
        self.__material = material

    def getGender(self):
        return self.__gender

    def setGender(self, gender):
        self.__gender = gender
